src/Pim/Bundle/CatalogBundle/Resources/config/query_builders.yml

```diff
@@ -10,6 +10,8 @@ parameters:
     pim_catalog.doctrine.query.filter.object_code_resolver.class: Pim\Bundle\CatalogBundle\Doctrine\Common\Filter\ObjectCodeResolver
     pim_catalog.doctrine.query.filter.dummy.class:              Pim\Bundle\CatalogBundle\Doctrine\Common\Filter\DummyFilter
 
+    pim_catalog.elastic_search.query.filter.string.class: Pim\Bundle\CatalogBundle\ElasticSearch\Filter\StringFilter
```

Comment:
<annoying_but_true_remark>
The official case for elastic search is "Elasticsearch" not "ElasticSearch" ;)
</annoying_but_true_remark>